kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java

```diff
@@ -106,14 +116,37 @@
     return tableIdsToTopics;
   }
 
-  /**
-   * Strips illegal characters from a table name. BigQuery only allows alpha-numeric and
-   * underscore. Everything illegal is converted to an underscore.
-   *
-   * @param tableName The table name to sanitize.
-   * @return A clean table name with only alpha-numerics and underscores.
-   */
-  private static String sanitizeTableName(String tableName) {
-    return tableName.replaceAll("[^a-zA-Z0-9_]", "_");
+  private static String getTopicToTableSingleMatch(BigQuerySinkConfig config, String topicName) {
```

Comment:
could there be a doc string for this method?

Reply:
Done.